.github/workflows/cpp.yml

```diff
@@ -43,10 +43,19 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install dependencies
         run: |
-          brew pin python@3.10
           cd $(brew --repository)
           git fetch origin --tags
           git checkout 3.6.16  # Arrow 10.0
+          rm -rf /usr/local/bin/2to3
+          rm -rf /usr/local/bin/2to3-3.11        
+          rm -rf /usr/local/bin/idle3          
+          rm -rf /usr/local/bin/idle3.11          
+          rm -rf /usr/local/bin/pydoc3
+          rm -rf /usr/local/bin/pydoc3.11
```

Comment:
This seems a lot, can `brew link --force` do ?

Reply:
I think this was the hint in the error which I tried first. Not sure why it didn't work.